pkg/serverconfig/validate.go

```diff
@@ -127,6 +132,18 @@ func validateProjectAccessClusterRolesJSON(value string) ([]string, error) {
 	return projectAccessOptions, nil
 }
 
+func validateI18nNamespaces(value string) error {
+	if value == "" {
+		return nil
+	}
+	matched, err := regexp.MatchString(`^\w[a-zA-Z0-9_\-,]*$`, value)
```

Comment:
This doesn't look right to me. For intsnace `foo,` would be allowed, and I'm not sure what the valid values are for an i18next namespace. I think it's easier to only try to split the string if it's not empty and then fail if any of the parts are `''`.